content/about/notes/1.0.1/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Istio 1.0.1
+weight: 91
+icon: /img/notes.svg
+---
+
+We're proud to release Istio 1.0.1! Within this release, we have been addressing some critical issues found by the community when using Istio 1.0.
```

Comment:
Unlike 1.0, this release is not such a big deal, so I would remove the first sentence: We’re proud to release Istio 1.0.1!

Reply:
are we not proud of it? :). i can remove it.